### PR TITLE
Add a setting to show actual mod count versus all mod count

### PR DESCRIPTION
**Pull request by AlexiyOrlov**
A small PR; adds a button to toggle the actual mod count (excludes libraries); when true, it will show like "Mods (1/29) loaded", otherwise will show as before.

### CODE REVIEW

**Comment by Mhowser** [comment]
This would be so nice to have!

**Comment by modmuss50** [comment]
I like this idea 👍

I will look at this in more deatail soon.

File: src/main/java/io/github/prospector/modmenu/ModMenu.java [hunk 1]
```diff
@@ -5,7 +5,9 @@
 import com.google.gson.FieldNamingPolicy;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.sun.org.apache.xpath.internal.operations.Mod;
```
Comment (shedaniel) [inline src/main/java/io/github/prospector/modmenu/ModMenu.java]: what is this

**Comment by Prospector** [comment]
Personally I find this to be unintuitive and confusing to have two numbers on the main menu, so I'd prefer to keep it simple.

**Pull request closed without merging**